.env.sample

```
@@ -0,0 +1,9 @@
+# docker-compose.yml variables
+
+CCACHE_VERSION=4.1
+CMAKE_VERSION=3.20.2
+CUDAARCHS=72-real
+CUDA_VERSION=11.0
+NODE_VERSION=15.14.0
+PARALLEL_LEVEL=8
+YARN_VERSION=1.22.5
```

Removing the ones I don't recommend people change:

```suggestion
CUDA_VERSION=11.0
PARALLEL_LEVEL=8
```

They _could_ change them, but we won't have tested them. And some like `YARN_VERSION` are tied to whatever ships in node's container, so it's implicitly tied to the `NODE_VERSION`.